src/sentry/static/sentry/app/views/settings/components/forms/field/fieldControl.tsx

```diff
@@ -11,7 +11,7 @@ const defaultProps = {
   flexibleControlStateSize: false,
 };
 
-type Props = typeof defaultProps & {
+type Props = Partial<typeof defaultProps> & {
```

Comment:
I think we can leave out `Partial` if we type `static defaultProps`, see https://www.notion.so/sentry/Sentry-Typescript-Tips-cef7ae63b4334156945b93b30a651b14

Reply:
@billyvg Tried following the notion doc, I still get an error saying `flexibleControlStateSize` is missing